NSString+MD5.h

```diff
@@ -7,8 +7,25 @@
 
 @import Foundation;
 
+
+#ifdef NS_ASSUME_NONNULL_BEGIN
+#define MD5_DIGEST_ASSUME_NONNULL_BEGIN NS_ASSUME_NONNULL_BEGIN
```

Comment:
Instead of defining this new macro can we just have

```
NS_ASSUME_NONNULL_BEGIN
```

inside this `ifdef`? Normally this would be great but since this library is so tiny we won't need to reuse this.